tools/python/run_android_emulator.py

```diff
@@ -43,8 +43,8 @@ def parse_args():
 
     args = parser.parse_args()
 
-    if not args.start and not args.stop:
-        # unspecified means start and stop
+    if not args.start and not args.stop and not args.create_avd:
+        # unspecified means start and stop if not creating the AVD
```

Comment:
nit: maybe also update the ArgumentParser description to say that only creating the AVD is also allowed